### PR TITLE
Remove "The" infection monkey

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do?

Removes "the" in front of "infection monkey"

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Comment by mssalvatore** [comment]
@triciakickssaas Is this in line with your standards for Infection Monkey?

**Review by cakekoa — COMMENTED** [review]
I found a handful of other instances of "the Infection Monkey", in addition to our main README:

https://github.com/guardicore/monkey/blob/fff08b67bf7dea184497e9174573858563a451df/docs/content/usage/scenarios/network-breach.md?plain=1#L24

https://github.com/guardicore/monkey/blob/fff08b67bf7dea184497e9174573858563a451df/docs/content/usage/scenarios/other.md?plain=1#L9-L10

[docs/layouts/shortcodes/homepage_shortcuts.html](https://github.com/guardicore/monkey/blob/the-infection-monkey/docs/layouts/shortcodes/homepage_shortcuts.html)

https://github.com/guardicore/monkey/blob/fff08b67bf7dea184497e9174573858563a451df/monkey/infection_monkey/readme.md?plain=1#L59

https://github.com/guardicore/monkey/blob/fff08b67bf7dea184497e9174573858563a451df/monkey/monkey_island/docs/source/index.rst?plain=1#L29

File: README.md
Comment (cakekoa) [inline README.md:1]: https://github.com/guardicore/monkey/blob/5c4cdd015a0a8e335114722741d08974777d4900/README.md?plain=1#L27

https://github.com/guardicore/monkey/blob/5c4cdd015a0a8e335114722741d08974777d4900/README.md?plain=1#L79

Comment (mssalvatore) [reply README.md:1]: Fixed.

**Review by cakekoa — APPROVED** [review]
LGTM

**Pull request merged**